src/Components/Divider.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {AbsoluteFill} from 'remotion';
+
+export const Divider: React.FC<{
+	style: React.CSSProperties;
```

Comment:
Style props must be optional for me, it's strange to have an error which said that `style props is required`

Reply:
Done